apparmor.d/profiles-g-l/git

```diff
@@ -18,6 +18,8 @@ profile git @{exec_path} flags=(attach_disconnected) {
   include <abstractions/nameservice-strict>
   include <abstractions/ssl_certs>
 
+  capability dac_read_search,
+
```

Comment:
This mostly depend of the location of the git repo. For normal location, it should not be needed.